eth_verkle/tree/commitment.nim

```diff
@@ -10,11 +10,12 @@
 ##  This module provides methods to generate commitments for tree nodes
 
 import
-  std/[tables, sequtils],
+  std/[tables, sequtils, strformat],
   elvis,
+  ../utils,
   ./tree,
   ../../constantine/constantine/hashes,
-  ../../constantine/constantine/serialization/[io_limbs, codecs_banderwagon, codecs],
+  ../../constantine/constantine/serialization/[io_limbs, codecs],
```

Comment:
ideally those imports ate all reexported from constantine/eth_verkle_ipa

Reply:
i think this should do, as the API has been upstreamed
```nim
 ../../constantine/constantine/ethereum_verkle_trees
```